docs/machine-learning/resources/tasks.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: Machine learning tasks
-description: Explore the different machine learning tasks supported in ML.NET.
+title: Machine learning tasks - ML.NET | Microsoft Docs
```

Comment:
don't add | Microsoft Docs. this is automatically added by the build

Reply:
oh, it got called out. Will remove!